web/src/components/OrganizationContentInfo.tsx

```diff
@@ -60,6 +60,7 @@ const OrganizationContentInfo = ({ organization }: OrganizationContentInfoProps)
               {{ domain: new URL(organization.url).hostname }}
             </a>
             by i18n
+            
```

Comment:
🙃  Please don't edit unrelated components, and if the changes there are removed from the PR over time, then please revert all changes in that file